common/extensions/api/brave_theme.json

```diff
@@ -56,11 +56,23 @@
               {
                 "name": "type",
                 "type": "string",
-                "description": "current theme type(Dark or Light)"
+                "description": "current theme type (Dark or Light)"
               }
             ]
           }
         ]
+      },
+      {
+        "name": "setBraveThemeType",
+        "type": "function",
+        "description": "Set current theme type",
+        "parameters": [
+          {
+            "name": "type",
+            "type": "string",
```

Comment:
@bsclifton I think using `integer` as a type here is much better and simpler for rest implementation.
And, that integer value would be `BraveThemeType`.
The reason is `braveTheme.getBraveThemeList` give the list of available pairs(`BraveThemeType` and its string)  by `BraveThemeService::GetBraveThemeList()` like `{ {value: 1, name: "Dark"}, {value: 2, name: "Light"} }`.

With this, welcome page just can use `value` for when calling `braveTheme.setBraveTheme`.

Reply:
Thanks for the feedback, @simonhong - I'll make a change to this today to use an int instead
1 = dark
2 = light
any other value = default

Reply:
Just so everyone has more context - according to the design spec, we want to display the dropdown options in a the following order (Light, Dark, System theme (default)) which diverges from how the dropdown in the settings menu is implemented

Currently the only usage we have for the `getBraveThemeList` is the response length so we know when or not to display `System theme (default)` - it's not being used to actually populate the dropdown😅IMO we can just leave the `setBraveTheme `as is since we don't need to couple it `getBraveThemeList`

Reply:
++ from me on leaving as-is, since the strings aren't used. Is that OK, @simonhong?

Reply:
ping @simonhong

Reply:
I see. I didn't know both use different option names. Thanks for explanation.